styleguide/_themes/global/scss/fonts.scss

```diff
@@ -2,6 +2,7 @@ $heading: 'Open Sans', Helvetica, Arial, sans-serif;
 $subhead: 'Montserrat', Helvetica, Arial, sans-serif;
 $copy: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 $code: Tahoma, Arial, Geneva, Helvetica;
+$semi-bold: 600;
```

Comment:
Ehh,  I'll probably scrap it.  I was originally planning on creating a variable that could be used across the board, but that probably should be done in another task, because there are other weights to take into consideration as well.